lib/parser/abiParser.ts

```diff
@@ -254,3 +254,38 @@ export function extractAbi(rawJson: string): RawAbiDefinition[] {
 
   throw new MalformedAbiError("Not a valid ABI");
 }
+
+export function extractBytecode(rawContents: string): string | null {
```

Comment:
Please use `undefined` instead of `null`. We should prefer undefined in our codebase (generated code is a different thing